src/verification.js

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {MatrixClientPeg} from './MatrixClientPeg';
+import dis from "./dispatcher";
+import Modal from './Modal';
+import * as sdk from './index';
+import { _t } from './languageHandler';
+import {RIGHT_PANEL_PHASES} from "./stores/RightPanelStorePhases";
+import {findDMForUser} from './createRoom';
+import {accessSecretStorage} from './CrossSigningManager';
+import SettingsStore from './settings/SettingsStore';
+
+export async function enable4SIfNeeded() {
+    const cli = MatrixClientPeg.get();
+    if (!cli.isCryptoEnabled() || !SettingsStore.isFeatureEnabled("feature_cross_signing")) {
+        return false;
+    }
+    const masterPK = cli.getCrossSigningId();
```

Comment:
There are some scenarios where we'll use key sharing to request the USK / SSK, and thus only have those around...

Perhaps `cli.getCrossSigningId("user_signing")` is a closer to what you need here...?